Js Session-1/js/script.js

```diff
@@ -0,0 +1,95 @@
+// on load hide field od showcity
+function init() {
+	document.getElementById('selectCity').style.display='none';	
+}
+
+// to change screen on anchor
+function changeScreen(target) {
+	    window.location.href = target+'.html';
+}
+
+$(document).ready(function(){
+    $('#mylist a').click(function(){
+        switch ($('#mylist a').index(this)) {
+			case 0: window.location.href = 'index.html';
+					break;
+			case 1:	alert("No page yet");
```

Comment:
Set this line window.location.href = '#'; other wise on click of these links page will throw error.